railties/lib/rails/test_help.rb

```diff
@@ -11,7 +11,7 @@
 
 require "active_support/testing/autorun"
 
-if defined?(Capbyara)
+if defined?(Capbyara) && defined?(Puma)
```

Comment:
It looks like there is a typo in the constant name ; this will never be true. And it looks like it has been replicated in the second change in your pull request.